### DISCUSSION TITLE
Can't detect Widevine after installation (version 119.0.6045.105)

### DISCUSSION THREAD

**Discussion by Mysticodex** [Eloston/ungoogled-chromium#2598] [General]
Widevine has been installed into my Windows's "Appdata/local/chromium/user data/widevinecdm/_platform_specific/win_x64/, i've restarted Chromium, but under Chrome Components the installed version of Widevine is still displaying as 0.0.0.0.

Any ideas what else I can try?

Thanks

**Comment by PF4Public** [comment]
Could be you need to update Widevine? Did it work before?

**Comment by Mysticodex** [comment]
It's never worked, this is a net new install of the files

**Reply by PF4Public** [reply]
Which binary you used for install? Not all of them might be built with Widevine support.

**Comment by Mysticodex** [comment]
Latest here:
https://ungoogled-software.github.io/ungoogled-chromium-binaries/releases/windows/64bit/

**Reply by PF4Public** [reply]
@Nifury Do these binaries support Widevine?

**Reply by Nifury** [reply]
Yes. 
It might be because Widevine was installed to a wrong location. 

![image](https://github.com/ungoogled-software/ungoogled-chromium/assets/7867461/d480716a-2c0c-4bb2-8daf-739324384586)
![image](https://github.com/ungoogled-software/ungoogled-chromium/assets/7867461/7425647a-41e5-425b-ae64-154828bbd462)
![image](https://github.com/ungoogled-software/ungoogled-chromium/assets/7867461/cfd1092a-80f4-437f-a2d2-8e0ee9a1c197)

**Reply by Mysticodex** [reply]
Bingo! That did it.

I missed the screenshot link on the instruction page for Widevine - works now :)

Thanks all
